plugins/amd-gpu/fu-amd-gpu-device.c

```diff
@@ -1,7 +1,16 @@
 /*
  * Copyright (C) 2023 Advanced Micro Devices Inc.
+ * All rights reserved.
  *
- * SPDX-License-Identifier: LGPL-2.1+
+ * This software and associated documentation (if any) is furnished
+ * under a license and may only be used or copied in accordance
```

Comment:
Do we need these three lines?

Reply:
it matches what is done in the dell-dock plugin for dual license.  legal pretty much said you can do the same thing but `s/Dell/AMD/`

Reply:
Right, I wondered if we could just add the `This file is provided under a dual MIT/LGPLv2 license` bit on the logic we want to avoid as much boilerplate as possible. If that's not possible, then no worries, and then LGTM.